tests/unit/sources/elasticsearch/test_api.py

```diff
@@ -421,6 +423,11 @@ def test_get_deployment(self: object, mock_query_hits: object) -> None:
         self.assertEqual(deployment.ip_version.value, '4')
         self.assertEqual(deployment.topology.value, '')
 
+    def test_spec_deployment(self: object):
```

Comment:
This tests the error that happens when there are more than on job, but what about the 'for' loop that follows it. That looks more important to test.